### PR TITLE
[User Roles] Disable Disbursement form fields for members

### PR DESCRIPTION
**Pull request by malted**
image

### CODE REVIEW

File: app/views/disbursements/_form.html.erb [hunk 1]
```diff
@@ -49,18 +53,20 @@
...
+    <%= form.text_field :name, placeholder: "Donating extra funds to another organization", maxlength: 60, required: true, disabled: %>
     <span class="muted">This is to help HCB keep record of our transactions. It will be used as the memo on your transaction list.</span>
   </div>
 
-  <div class="actions inline-block">
-    <%= form.submit "Make transfer", class: "btn", disabled: @source_event&.demo_mode? %>
-  </div>
+  <%= content_tag :div,
+    class: "actions inline-block #{"tooltipped tooltipped--e" if disabled}",
+    aria: { label: playground_mode ? "Disbursements are disabled in Playground mode" : !user_can_transfer ? "Only managers can make disbursements!" : nil } do %>
```
Comment (cjdenio) [inline app/views/disbursements/_form.html.erb]:
```suggestion
    aria: { label: playground_mode ? "Transfers are disabled in Playground mode" : !user_can_transfer ? "Only managers can make transfers!" : nil } do %>
```

**Comment by sarthaktexas** [comment]
i just KNOW gary is going to say move this logic into the controller

**Comment by garyhtou** [comment]
> i just KNOW gary is going to say move this logic into the controller

LOL, you're starting to get the hang of it!

<br/><br/>

Close, i'd suggest moving the `@source_event&.demo_mode?` check into the policy, and then just using
```ruby
!policy(@source_event).new_transfer?
```
to control the disabled attribute.

However, I'm cool with how it currently is; I'm focusing on the priority of fixing bugs and getting this out to users ASAP

File: app/views/disbursements/_form.html.erb [hunk 2]
```diff
@@ -49,18 +51,20 @@
...
   </div>
 
-  <div class="actions inline-block">
-    <%= form.submit "Make transfer", class: "btn", disabled: @source_event&.demo_mode? %>
-  </div>
+  <%= content_tag :div,
+    class: "actions inline-block #{"tooltipped tooltipped--e" if disabled}",
+    aria: { label: playground_mode ? "Transfers are disabled in Playground mode" : !user_can_transfer ? "Only managers can make transfers!" : nil } do %>
+    <%= form.submit "Make transfer", class: "btn", disabled: %>
+  <% end %>
```
Comment (garyhtou) [inline app/views/disbursements/_form.html.erb]: i'm not sure if i understand why this needs to be a `content_tag`. i think w can use a normal `div`

**Review by garyhtou — APPROVED** [review]

**Pull request merged**